### PR TITLE
Additional code viewer tests

### PR DESCRIPTION
**Pull request by beckerhe**
Some additional tests for SourceCodeReport and DisassemblyReport

### CODE REVIEW

**Review by antonrohr — APPROVED** [review]
I have nothing to complain about

File: src/CodeReport/AssemblyTestLiterals.h [hunk 1]
```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <string_view>
+
+namespace orbit_code_report_internal {
```
Comment (dpallotti) [inline src/CodeReport/AssemblyTestLiterals.h:7]: Why the `_internal` here? This is a private header.

File: src/CodeReport/DisassemblerTest.cpp [hunk 2]
```diff
@@ -1,80 +1,35 @@
...
-    "0x40104f:\tlea          eax, [rbp + rbx]\n"
-    "0x401053:\tpop          rdx\n"
-    "0x401054:\tpop          rbx\n"
-    "0x401055:\tpop          rbp\n"
-    "0x401056:\tret          \n"
-    "0x401057:\tadd          dword ptr [rdi], ebx\n"  // Line 24
-    "0x401059:\tadd          byte ptr [rax], 0\n"
-    "0x40105c:\tadd          byte ptr [rax], al\n"
-    "0x40105e:\n\n"sv;  // Line 27 & 28
+using orbit_code_report_internal::kFibonacci;
```
Comment (dpallotti) [inline src/CodeReport/DisassemblerTest.cpp:10]: I would suggest `kFibonacciAssembly` or something like that for more clarity.

File: src/CodeReport/SourceCodeReportTest.cpp [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+  orbit_client_data::ThreadSampleData sample_data{};
+
+  SourceCodeReport report{"", function_info, 0x8000, &elf_file, sample_data, 0ul};
+
+  EXPECT_EQ(report.GetNumSamples(), 0);
+  EXPECT_EQ(report.GetNumSamplesInFunction(), 0);
+  EXPECT_FALSE(report.GetNumSamplesAtLine(0).has_value());
+}
+
+TEST(SourceCodeReport, StaticLineInfo) {
```
Comment (dpallotti) [inline src/CodeReport/SourceCodeReportTest.cpp:62]: Likely a stupid question, but what does "static" mean in this context?

**Review by dpallotti — APPROVED** [review]
(I'm not really familiar with the code under test.)

File: src/CodeReport/AssemblyTestLiterals.h (hunk 1 above)
Comment (beckerhe) [reply src/CodeReport/AssemblyTestLiterals.h:7]: I don't really care. I will change it. :-)

File: src/CodeReport/SourceCodeReportTest.cpp (hunk 3 above)
Comment (beckerhe) [reply src/CodeReport/SourceCodeReportTest.cpp:62]: Not a stupid question. Makes not really sense since everything is static. I will change it.

Comment (beckerhe) [reply src/CodeReport/SourceCodeReportTest.cpp]: Done.

File: src/CodeReport/DisassemblerTest.cpp (hunk 2 above)
Comment (beckerhe) [reply src/CodeReport/DisassemblerTest.cpp]: Done.

File: src/CodeReport/AssemblyTestLiterals.h (hunk 1 above)
Comment (beckerhe) [reply src/CodeReport/AssemblyTestLiterals.h]: Done.

**Pull request merged**